.github/workflows/cibuildwheel.yml

```diff
@@ -69,9 +69,9 @@ jobs:
         run: twine check dist/*
 
       - name: Upload sdist result
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: sdist
+          name: cibw-sdist
```

Comment:
Ah @QuLogic sorry I didn't review before commenting, but given that you've given distinct names I think everything is fine as you're doing what @henryiii did in https://github.com/scientific-python/cookie/pull/350/files. So I think this is good (without having done it myself). 👍